pkg/updatestrategy/aws_ec2.go

```diff
@@ -0,0 +1,231 @@
+package updatestrategy
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/zalando-incubator/cluster-lifecycle-manager/provisioner"
```

Comment:
This looks like a circular dependency. I imagine that is why the build is failing?